update changelog.md (#730)
scale-it/algo-builder: CHANGELOG.md

- Add Runtime.getAppByName(appName): get app by name declared in appDefinition.
- For Algob.balanceOf(deployer, accountAddr, assetID) if assetID is undefined then function will return ALGO account balance.
- Add new example [Trampoline](https://github.com/algorand-devrel/demo-avm1.1/tree/master/demos/trampoline)

> - Add Txn LastLog

Reviewer: please add more context here, eg: _runtime: Txn LastLog opcode support_

> - Better warning/error when deploy ASA.

Reviewer: please add more details here

- Add Runtime.getAppByName(appName): get app by name declared in appDefinition.
- For Algob.balanceOf(deployer, accountAddr, assetID) if assetID is undefined then function will return ALGO account balance.
- Add new example [Trampoline](https://github.com/algorand-devrel/demo-avm1.1/tree/master/demos/trampoline)
- Runtime: add `Txn LastLog` opcode.

> - Runtime: better warning/error when deploy ASA. Throw error when get ASA definition file or ASA not found in asa.yaml but Runtime need query data from it.

Reviewer: ```suggestion
- Runtime: better warning/error when deploying ASA. Throws an error when ASA definition is wrong or when ASA is not found in asa.yaml, eg when Runtime needs to query ASA.
```
